src/nunit-gui/nunit-gui.csproj

```diff
@@ -34,15 +34,23 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Mono.Cecil, Version=0.9.6.0, Culture=neutral, PublicKeyToken=0738eb9f132ed756, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\NUnit.Engine.3.6.0-dev-03383\lib\Mono.Cecil.dll</HintPath>
+      <HintPath>..\..\packages\NUnit.Engine.3.6.1\lib\Mono.Cecil.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="nunit.engine, Version=3.6.0.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
-      <HintPath>..\..\packages\NUnit.Engine.3.6.0-dev-03383\lib\nunit.engine.dll</HintPath>
+    <Reference Include="nunit-agent, Version=3.6.1.0, Culture=neutral, processorArchitecture=MSIL">
```

Comment:
This is done for both assemblies. Probably, since I forgot about it, the process needs to be documented somewhere. Given that it's a temporary situation (we hope) do you think it's worth adding a text file in the project root? Or should it go into the wiki docs?

FYI, my plan is that the engine should eventually be in the package tools subdirectory, with only a copy of the api assembly present in the lib directory. That way, nothing else will get referenced by the gui.